src/main/kotlin/org/jitsi/nlj/rtcp/KeyframeRequester.kt

```diff
@@ -177,7 +186,7 @@ class KeyframeRequester(
 
     override fun getNodeStats(): NodeStatsBlock {
         return super.getNodeStats().apply {
-            addString("wait_interval_ms", waitIntervalMs.toString()) // use string to prevent aggregation
+            addNumber("wait_interval_ms", waitInterval.toMillis())
```

Comment:
I don't think we want to aggregate these (it will be confusing if we look at the output of the HTTP API and see aggregated values there), so it should still be `addString`